keygen/openssl.conf

```diff
@@ -1,4 +1,35 @@
 [req]
 distinguished_name = req_distinguished_name
+x509_extensions = v3_ca	# The extentions to add to the self signed cert
```

Comment:
extentions -> extensions
Please avoid tabs. The comment should be above that line, not at the end.